### DISCUSSION TITLE
using keyword combined with discard in out params

### DISCUSSION THREAD

**Discussion by henke37** [dotnet/csharplang#4595] [Language Ideas] [answered]
The `using` keyword is awesome, especially when working with native objects that would normally need a finalizer.

The discard "variable" is great when working with `out` parameters that you don't care about.

However, you can't combine the two to elegantly dispose an object that you just want to discard.

It'd be cool if you could write a line like this:
```csharp
Fun(using out _);
```

**Comment by HaloFour** [comment, marked as answer]
See: https://github.com/dotnet/csharplang/discussions/3544
